README.md

```diff
@@ -1,4 +1,4 @@
-# Kworkflow
+[![kworkflow](images/kw_logo.svg)](https://github.com/kworkflow/kworkflow/)
```

Comment:
Why not use png? Additionally, how about use kw instead of kworkflow? We'll change it soon.